code/game/objects/items/kitchen.dm

```diff
@@ -144,6 +145,25 @@
 	righthand_file = 'icons/mob/inhands/equipment/kitchen_righthand.dmi'
 	w_class = WEIGHT_CLASS_NORMAL
 
+/obj/item/kitchen/knife/bloodletter
+	name = "bloodletter"
+	desc = "An occult looking dagger that is cold to the touch. Somehow, the flawless orb on the pommel is made entirely of liquid blood."
+	icon = 'icons/obj/ice_moon/artifacts.dmi'
+	icon_state = "bloodletter"
+	w_class = WEIGHT_CLASS_NORMAL
+	/// Bleed stacks applied when an organic mob target is hit
+	var/bleed_stacks_per_hit = 3
+
+/obj/item/kitchen/knife/bloodletter/attack(mob/living/M, mob/living/carbon/user)
+	. = ..()
+	if(!istype(M) || !(M.mob_biotypes & MOB_ORGANIC))
+		return
+	var/datum/status_effect/stacking/saw_bleed/bloodletting/B = M.has_status_effect(/datum/status_effect/stacking/saw_bleed/bloodletting)
+	if(!B)
+		M.apply_status_effect(/datum/status_effect/stacking/saw_bleed/bloodletting, bleed_stacks_per_hit)
+	else
+		B.add_stacks(bleed_stacks_per_hit)

```

Comment:
This will happen even if the attacker is pacifist and the attack doesn't happen.
Is that intended?